pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1658,8 +1654,7 @@
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
         "4.1.0.0": "4.1.0",
-        "4.1.1.0": "4.1.1",
-        "4.2.0.0": "4.2.0",
+        "4.1.1.0": "4.3.0",
```

Comment:
Not in master, no.  I believe this was from when Steve initially did some API additions in master then reverted them.  I picked it up in my first index creation since it happened to be in my corefx package cache.